frontends/mit-open/src/pages/FieldPage/FieldPageSkeleton.tsx

```diff
@@ -0,0 +1,186 @@
+import React from "react"
+import { Link } from "react-router-dom"
+import * as routes from "../../common/urls"
+import { BannerPage, styled, Container } from "ol-components"
+import { useFieldDetail } from "api/hooks/fields"
+import FieldMenu from "@/page-components/FieldMenu/FieldMenu"
+import FieldAvatar from "@/page-components/FieldAvatar/FieldAvatar"
+
+export const FieldStyles = styled.div`
+  .field-title-container.MuiContainer-root {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100%;
+  }
+
+  .field-title-row {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    img {
+      height: $avatar-size;
+      width: $avatar-size;
+    }
+
+    h1 a {
+      color: theme.$color-gray-1;
```

Comment:
**Request:** I think we should remove this and all other $ variables. Either replace them (like suggested below) or just delete.


All of the `$` variables are old sass variables and won't work with Emotion. I would suggest either using `white`, or you can access the theme via
```js
color: ${({ theme }) => theme.custom.colorGray1};
```

BTW: I forget if you use VS Code, but https://marketplace.visualstudio.com/items?itemName=styled-components.vscode-styled-components is a good extension to install if you do. (It says styled components, which is a library similar to Emotion. The extension works well with both.)